extensions/dashlane-vault/README.md

```diff
@@ -29,8 +29,32 @@ brew install dashlane/tap/dashlane-cli
 
 Follow the authentication steps from the official [Dashlane CLI documentation](https://dashlane.github.io/dashlane-cli/personal/authentication).
 
+### Step 3 - Healtcheck
+
+If the extension is not working as expected check out the **Settings** and **Troubleshooting** sections.
+
 ### You're all set! 🎉
 
+## Settings
+
+### CLI Path
+
+Retrieve the path to the Dashlane CLI by running the following command in your terminal:
+
+```sh
+which dcli
+```
+
+Set the path to the Dashlane CLI in the extension preferences of Raycast. Open Raycast preferences, navigate to the Dashlane Vault extension, and specify the path to the Dashlane CLI. You can also use the following shortcut to jump directyl to the extenions settings: `⌥ + ⌘ + ,`
+
+### Master Password
```

Comment:
Could add a short explanation on how to use biometrics unlock?

Reply:
Currently this pull request does not introduce this feature to the extension yet. I tried it and it mostly worked. 

But I noticed that if you set `save-master-password` to false and `user-presence` to biometric you can no longer use the master password within the environment variables. So you will have to insert the password after the biometric successfully was requested, which is not possible by using it via Nodejs. Could you confirm this @Mikescops?

But I will add the possibility to use either save-master-password or biometrics within this pull request.

Reply:
I tried `save-master-password` to false and `user-presence` to biometrics and passing the env variable `DASHLANE_MASTER_PASSWORD` works just fine for me.

Also I'm not sure to understand why you put a condition on biometrics to open the raycast app?

Reply:
> I tried save-master-password to false and user-presence to biometrics and passing the env variable DASHLANE_MASTER_PASSWORD works just fine for me.

I checked it again and it is working as you mentioned. @Mikescops 

> Also I'm not sure to understand why you put a condition on biometrics to open the raycast app?

If the biometrics prompt is shown the raycast app is not visible anymore. After the prompt was successful, I want to show the raycast app again.

Reply:
Alright good for me